### DISCUSSION TITLE
findAllActiveListingsByShop to include images

### DISCUSSION THREAD

**Discussion by Topazjames** [etsy/open-api#1032] [Ideas]
It would be useful for the findAllActiveListingsByShop endpoint to include images, like how you can on getListingsByShop.

on a website I'm building, I'm having to do 2 separate API calls - one to get all the featured listings, I then can get a list of listing ID's, and can then do a call to the getListingsByListingIds to get the exact same data but this time with images attached.

**Comment by mgeric** [comment]
Yes, we need this too! Crazy having to make two separate calls only to get exact same data + images the second time.

**Reply by emergingdzns** [reply]
This has been requested a bunch of times. Doesn't seem to be a priority with Etsy.

What we have done is to call `findAllActiveListingsByShop` and then get the list of listing_ids from that response. Then make ONE call back to `getListingsByListingIds` with Images in the includes.
